flow/designs/nangate45/swerv_wrapper/rules.json

```diff
@@ -1,49 +1,14 @@
 {
     "rules": [
-        {
-            "field": "synth__area__stdcell__count",
-            "value": 129648,
-            "compare": "<="
-        },
         {
             "field": "constraints__clocks__count",
             "value": 1,
             "compare": "=="
         },
-        {
-            "field": "placeopt__area__instance__area",
-            "value": 583842,
-            "compare": "<="
-        },
-        {
-            "field": "placeopt__area__instance__count",
-            "value": 143020,
-            "compare": "<="
-        },
         {
             "field": "globalroute__timing__clock__slack",
-            "value": -0.27,
+            "value": -0.29,
             "compare": ">="
-        },
-        {
-            "field": "detailedroute__wirelength",
-            "value": 5939277,
-            "compare": "<="
-        },
-        {
-            "field": "detailedroute__errors__count",
-            "value": 0,
-            "compare": "=="
-        },
-        {
-            "field": "detailedroute__drc__error__count",
-            "value": 0,
-            "compare": "=="
-        },
-        {
-            "field": "finish__area",
-            "value": 2262308,
-            "compare": "<="
```

Comment:
This is related to the [OR-998](https://openroad.atlassian.net/browse/OR-998) -- i.e., the design was failing and we could not generate a new baseline. Since Cherry already fixed this, @ravi-varadarajan could you run this design again (maybe using OpenROAD master) to get an ok and rules file? Or you can commit using [these artifacts](https://jenkins.openroad.tools/blue/organizations/jenkins/OpenROAD-flow-scripts-Nightly-Public/detail/OpenROAD-flow-scripts-Nightly-Public/45/artifacts) from CI.

Reply:
fixed on 2e36130